clients/da-vinci-client/src/test/java/com/linkedin/davinci/kafka/consumer/StoreIngestionTaskTest.java

```diff
@@ -624,6 +629,16 @@ public void methodSetUp() throws Exception {
     doReturn(regionStats).when(kafkaConsumerServiceStats).getStoreStats(anyString());
   }
 
+  private InMemoryPubSubPosition convertToInMemoryPosition(Object position) {
+    if (position instanceof InMemoryPubSubPosition) {
+      return (InMemoryPubSubPosition) position;
+    } else if (PubSubSymbolicPosition.EARLIEST.equals(position)) {
+      return InMemoryPubSubPosition.of(-1L);
```

Comment:
Should we use `EARLIEST.getNumericOffset` here and   `LATEST.getNumericOffset` below?

Reply:
I just wanted to minimize the use of `getNumericOffset`, and these symbolic positions do not provide `getInternalOffset` also.
If we use `getNumericOffset`, at some point we will have to remove it.

Reply:
Ok. We could consider adding variation of `PubSubUtil::computeOffsetDelta` . Maybe in different PR